mne_bids/write.py

```diff
@@ -1164,7 +1168,10 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
     if events is not None and len(events) > 0 and not emptyroom:
         _events_tsv(events, raw, events_fname, event_id, overwrite, verbose)
 
-    make_dataset_description(bids_root, name=" ", verbose=verbose)
+    dataset_description_fpath = op.join(bids_root, "dataset_description.json")
+    if not op.exists(dataset_description_fpath) or overwrite:
+        make_dataset_description(bids_root, name=" ", verbose=verbose)
```

Comment:
humm I didn't realize this was already being created in write_raw_bids. Can you update the test from the PR of @sappelhoff : https://github.com/mne-tools/mne-bids/pull/342/files